### PR TITLE
[JA] Translate Basics/Basics

### PR DESCRIPTION
**Pull request by penpenpng**
Added translation for basics/basics.

### CODE REVIEW

File: ja/lessons/basics/basics.md [hunk 1]
```diff
@@ -211,6 +211,8 @@ iex> not 42
 ** (ArgumentError) argument error
 ```
 
+参考: Elixirの`and`と`or`はErlangの`andalso`と`orelse`に実際に対応しています。
```
Comment (riseshia) [inline ja/lessons/basics/basics.md]: don't we need to translate `which doesn't evaluates first argument and evaluates the second argument.` ?

**Comment by riseshia** [comment]
cc @elixirschool/japanese

File: ja/lessons/basics/basics.md (hunk 1 above)
Comment (marocchino) [reply ja/lessons/basics/basics.md]: removed? https://github.com/elixirschool/elixirschool/commit/22a033dc7fe90fe9a49e51983b7ab493b7246b43#diff-56389d258534fde2065909c908684202

Comment (riseshia) [reply ja/lessons/basics/basics.md]: oops

**Comment by riseshia** [comment]
Thanks you for your contribution!!!

**Pull request merged**